rosette/src/protobuf/Ob.proto

```diff
@@ -162,7 +197,7 @@ enum ObType {
 
 message Object {
     ObType type = 1;
-    uint64 id = 2;
+    uint64 objectId = 2;
```

Comment:
Please follow the protocol buffer style guide unless there's a really good reason not to.  In this file, three different naming conventions have been used.  The generated code for C++ converts `foo_bar_baz` to `fooBarBaz`; we should use all lowercase and underscore between words.
https://developers.google.com/protocol-buffers/docs/style

Reply:
Thank you for pointing me to this style guide.  If it's okay with you, i'd like to merge this PR with the existing names, and then create a separate PR that implements the style guide. I see several other things from the guide that should be changed, and I'd like to do them all at once.

Reply:
OK.